src/logic/month-copy/month-copy.logic.ts

```diff
@@ -101,23 +103,14 @@ function concatWithLastWeekFromPrevMonth<T>(
   prevMonth: T[],
   currentMonth: T[]
 ): T[] {
-  const prevMonthLastWeek = MonthHelper.getMonthLastWeekData(monthKey, prevMonth, currentMonth);
-  return prevMonthLastWeek ? prevMonthLastWeek.concat(copiedData) : copiedData;
+  return MonthHelper.getMonthLastWeekData(monthKey, prevMonth, currentMonth).concat(copiedData);
 }
-
+//zawsze 28 lub 35 dni
 function getNumberOfDaysToBeCopied(monthKey: ScheduleKey): number {
-  const {
-    daysMissingFromNextMonth: daysEditedInPrevMonth,
-  } = MonthHelper.calculateMissingFullWeekDays(monthKey.prevMonthKey);
-  const { daysMissingFromNextMonth: daysFromNextMonth } = MonthHelper.calculateMissingFullWeekDays(
-    monthKey
-  );
-
-  return (
-    MonthHelper.getMonthLength(monthKey.year, monthKey.month) -
-    daysEditedInPrevMonth +
-    daysFromNextMonth
-  );
+  return MonthHelper.findFirstMonthMondayIdx(monthKey.year, monthKey.month) + 28 >=
```

Comment:
To mi się bardzo podoba, dobrze przemyślane :D

Reply:
Moim zdaniem poprzednia implementacja jest czytelniejsza, ponieważ lepiej widać logikę która stoi za wyznaczaniem tego ile dni my będziemy kopiować na następny miesiąc.

Zgadzam się z tym, że twoja wersja jest krótsza, ale patrząc na nią osobie które nie jest temacie będzie trudno zrozumieć dlaczego dozwolone wartości to tylko 28 albo 35.

Natomiast w tej implementacji to jest krok po kroku rozpisane.
Ponadto, w przypadku jeśli w jakiś sposób zmieni się logika kopiowania, to prościej będzie poprawić to w poprzedniej imeplementacji

Reply:
Jeden rabin powie tak, drugi rabin powie nie XD. 
U mnie była zależnośc od innych funkcji, w których mogą być błędy, więc lepiej postawmy na prostotę i rozwiązanie 
@kopcion.
Zgadzam się z tym, że 28 i 35 mogą być niejasne. Dlatego proponuję, żeby je dodać do opisowych stałych, żeby od razu było jasne skąd są te wartosci. Ewentualnie jeszcze jakiś komentarz dlaczego akurat tak.

Reply:
> U mnie była zależnośc od innych funkcji,

Nie wiem czy to jest złe. Przecież my używamy ich też w innych miejscach, więc jak będzie jakiś problem to aplikacja się wysypie, ale w innym miejscu. 

Ponadto, jeśli będziemy opisywać to w komentarzach to może być tak, że ktoś w jakiś sposób to zmieni, ale komentarz zostanie taki  jak był (ponieważ tego zwykłe nikt nie edytuje)

Więc ja dalej jestem za tym, żeby zostawić to tak jak było. Ale nie chcę narzucać. 
@kopcion 
Jak twoim zdaniem będzie lepiej biorąc pod uwagę to wszystko co jest wyżej?

Reply:
Powiem tak, nie będę naciskać. Jednak według mnie dużą zaletą mojej implementacji jest to że widać na pierwszy rzut oka ze zwraca tylko jedną z dwóch wartości. 
'ale patrząc na nią osobie które nie jest temacie będzie trudno zrozumieć dlaczego dozwolone wartości to tylko 28 albo 35' tu mogę tylko powiedzieć że osobie która nie jest w temacie jeszcze ciężej będzie zobaczyć że aktualna implementacja rzeczywiście zwraca tylko 28 albo 35

Reply:
ok